packages/blockchain-link/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.1.31-beta.1
+
+-   ci: automate connect version table generation (a7089f7)
```

Comment:
hmm this looks like it is picking changes from incorreect packages.

Reply:
There is an issue with the part of the script that check what packages need to be released, I am investigating. 
It is interesting because same script locally produces right results and in CI it is different. :thinking: